criu/cr-check.c

```diff
@@ -1266,6 +1266,14 @@ static int check_kcmp_epoll(void)
 	return 0;
 }
 
+static int check_time_namespace(void)
+{
+	if (!kdat.has_timens)
+		return -1;
+
+	return 0;
+}
+
```

Comment:
Nit: as kerndat is cached, you probably want to print here something if timens is unsupported by kernel.
Also, I think it's worth to call this in `cr_check()` for checking it in `criu check --all`